symengine/tests/printing/test_printing.cpp

```diff
@@ -782,275 +782,277 @@ TEST_CASE("test_unicode()", "[unicode]")
     std::string s;
 
     s = unicode(*complexes());
-    CHECK(s == "\u2102");
+    CHECK(s == u8"\u2102");
```

Comment:
How about,
```suggestion
    CHECK(s == std::string(u8"\u2102"));
```
?

Reply:
Thanks for the suggestion. I can try it.

Reply:
Turns it this doesn't work. C++20 cannot initialize `std::string` with u8-literals.